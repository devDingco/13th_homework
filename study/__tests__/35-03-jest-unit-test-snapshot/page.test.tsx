import { render } from "@testing-library/react";
import JestUnitTestSnapshotPage from "@/app/section35/35-03-jest-unit-test-snapshot/page";
import "@testing-library/jest-dom";

it("기존 사진이랑 바뀐게 없는지 비교해보자! - 스냅샷 테스트", () => {
  const result = render(<JestUnitTestSnapshotPage />);
  expect(result.container).toMatchSnapshot();
});
