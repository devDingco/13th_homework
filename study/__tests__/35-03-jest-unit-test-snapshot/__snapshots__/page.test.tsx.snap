// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진이랑 바뀐게 없는지 비교해보자! - 스냅샷 테스트 1`] = `
<div>
  <h1>
    철수는 14살 입니다.
  </h1>
  <div>
    철수의 취미 입력하게: 
    <input
      type="text"
    />
  </div>
  <button>
    철수랑 놀러가기
  </button>
</div>
`;
